shotover-proxy/examples/windsock/cassandra.rs

```diff
@@ -135,7 +233,7 @@ impl Bench for CassandraBench {
         &self,
         flamegraph: bool,
         _local: bool,
-        runtime_seconds: u32,
+        _runtime_seconds: u32,
```

Comment:
We need to handle this value properly so that `--operations-per-second` continues to work.
Also make sure that we are sending `Report::SecondPassed`